rome/src/main/java/com/rometools/rome/feed/synd/impl/ConverterForRSS091Userland.java

```diff
@@ -166,6 +166,11 @@ protected SyndEntry createSyndEntry(final Item item, final boolean preserveWireI
 
         syndEntry.setComments(item.getComments());
 
+        // DublinCoreTest will be failed without this row
+        // I don't have better solution
```

Comment:
It looks like you didn't figure it out? I mean, why exactly the test fails and whether it means that something is broken for other use cases.

Reply:
It's little dirty solution for this problem but i think it's shouldn't broke anything else. This problem need more deep solution for understanding cross-version syntax